### ISSUE TITLE
query.staking.bonded Bonded expect one argument

### ISSUE THREAD

**Issue by Tbaut** [polkadot-js/api#1069]
In Staking>actions

![image](https://user-images.githubusercontent.com/33178835/60516600-60aaf480-9cde-11e9-9350-6b684dfd8364.png)

This must be a problem from my recent PR to introducing multi and ParamPick here: https://github.com/polkadot-js/apps/blob/master/packages/app-staking/src/Actions/Accounts.tsx#L130

**Comment by Tbaut** [comment]
It works well in JS console ([example](http://localhost:3000/#/js/share/qAPwby8vIEFsbCBjb2RlIGlzIHdyYXBwZWQgd2l0aGluIGFuIGFzeW5jIGNsb3N1cmUsCi8vIGFsbG93aW5nIGFjY2VzcyB0byBhcGksIGhhc2hpbmcsIGtleXJpbmcsIHR5cGVzLCB1dGlsLgovLyAoYXMBTQQoe4oxABwgfSkgPT4gewFyOCAgLi4uIGFueSB1c2VyIBGtMGV4ZWN1dGVkIGhlcmUBIwEs8FJ9KSgpOwpjb25zdCBBbGljZVN0YXNoQWRkcmVzcyA9ICc1R05KcVRQeU5xQU5Ca1VWTU4xTFBQcnhYbkZvdVdYb2Uyd05TbW1Fb0xjdHhpWlknCglNNGJvbmRlZCA9IGF3YWl0AcI4LnF1ZXJ5LnN0YWtpbmcuCSEEKEE+dQAIKSA7BZAYb2xlLmxvZ0ohADIfAAgnaXMRbCh0bycsYm9uZGVkKQ==) not throwing using the same node that's throwing in Staking>Actions), and the whole Staking>Actions works as well, so it's not an `api` issue.

I thought the problem would rely in the fact that we are calling `query.staking.bonded` with a multi and this newly introduced `ParamPick`. The array we pass to `withCall` is probably empty at some point or not formed the way we expected.

```
      paramPick: ({ allAccounts }: Props) => {
        return allAccounts && Object.keys(allAccounts);
      },
```
I've been logging things in `withCall` extensively and couldn't spot a case where we indeed call with an empty array.. and @Stefie seems to have found it.

**Comment by Stefie** [comment]
>  @Stefie seems to have found it.

No, you're right. I was too quick to move it. The mistake was on my side, it does indeed work outside of apps.
